riva/clients/asr/streaming_recognize_client.cc

```diff
@@ -241,9 +242,19 @@ StreamingRecognizeClient::DoStreamingFromFile(
     std::cout << std::flush;
     double diff_time = std::chrono::duration<double, std::milli>(current_time - start_time).count();
 
+    float total_processed = 0.F;
+    if (all_wav[0]->encoding == nvidia::riva::AudioEncoding::OGGOPUS) {
+      riva::utils::opus::Decoder decoder;
+      std::ifstream is(all_wav[0]->filename);
+      auto wav_stream = decoder.DecodeStream(is);
+      total_processed = (float)wav_stream.size() / 48000.F;
```

Comment:
This just uses the length of the first file right?
I think we need to get the total number of opus samples from each wav is all_wav, and then compute the total time from that.

Reply:
No, we can't. OPUS is compressed. To get the number of PCM samples, we need to decompress. Which is OK because this decompression happens after after everything else, thus it doesn't affect timings reported,

Reply:
Sure, but this is still just the PCM sample of one file right?  What happens when I load a directory of files?

Reply:
Ah, good catch, thx. Fixed